libcontainer/network_linux.go

```diff
@@ -171,7 +171,7 @@ func (v *veth) create(n *network, nspid int) (err error) {
 			netlink.LinkDel(veth)
 		}
 	}()
-	if err := v.attach(&n.Network); err != nil {
+	if err = v.attach(&n.Network); err != nil {
```

Comment:
Ahh... I thought the `err` variable was the one returned, after checking the Golang grammar I think I was wrong. Actually the origin code is correct. This change fixed nothing. Sorry to misunderstand.